src/frontCANnode/inc/config.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _CONFIG_H_
+#define _CONFIG_H_
+
+/**
+ * TODO this file really should not need to exist.
+ * Is there a way to only include certain files from evt_lib
+ * so that the project does not need to implicitly include
+ * the ltc6804 drivers? ltc6804.c has a constraint expecting this
+ * file with these defines, which doesn't make sense for this project.
+ */
+
+#define MAX_NUM_MODULES 15
+#define MAX_CELLS_PER_MODULE 12
```

Comment:
Hm, the CAN node is monitoring a battery? :P (no immediate action needed, just joking)